.chronus/changes/feature-object-literals-2024-2-15-18-36-3.md

```diff
@@ -0,0 +1,17 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
+packages:
+  - "@typespec/compiler"
+---
+
+New Language Feature: Object and Tuple Literals.
```

Comment:
I thought we wanted to call these array literals?

Reply:
yeah I think this is from before, I haven't update those docs